demos/checkbox.scss

```diff
@@ -17,4 +17,24 @@
 @import "./common";
 @import "../packages/mdc-checkbox/mdc-checkbox";
 @import "../packages/mdc-form-field/mdc-form-field";
+@import "../packages/mdc-theme/color_palette";
 @import "../packages/mdc-typography/mdc-typography";
+
+$demo-custom-color: $material-color-red-500;
```

Comment:
Good call. It might not be a problem right now, but I plan to eventually reuse component demo styles in multiple places (the component demo page and the theme demo page), at which point it will become a problem. Fixed!